db/models/Deck.js

```diff
@@ -5,6 +5,7 @@ const { Schema } = mongoose;
 const deckSchema = new Schema({
   name: { type: String, required: true },
   color: { type: String, required: true, default: "#d56f21ff" },
+  reversedDirection: { type: Boolean, required: true, default: false },
```

Comment:
Your key "reversedDirection" is a boolean, I suggest to use the naming convention for boolean "is..." (isReversedDirection in your case) like you already did with other keys. If you do so, you would have to rename it everywhere you use it.